linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#Setup arguments
+
+#Main code
+#check if docker_deamon is running - otherwise start run
+systemctl status docker || systemctl start docker
+
```

Comment:
please review the shell style guide and fix your code
https://www.notion.so/jarviscanada/Shell-Style-Guide-764a3ac0c9044fdb844b55420d1c7594

Reply:
not sure about this, will check with edward tomorrow during office hours.